### DISCUSSION TITLE
How to add a banner between the header and content?

### DISCUSSION THREAD

**Discussion by syeda-git** [squidfunk/mkdocs-material#3064] [Q&A] [answered]
Hi,
The list of blocks on this [page ](https://squidfunk.github.io/mkdocs-material/customization/?h=base+html#overriding-blocks-recommended) do not add a banner between the header and content. The `extrahead` block adds a banner on top of the header. 

Is there a way to add a banner after the header?

PS: I have recently adopted MkDocs and Material so pardon my ignorance.

Syed

**Comment by squidfunk** [comment, marked as answer]
The `hero` block is what' you're looking for.

**Reply by syeda-git** [reply]
Thank you.

**Reply by rangulvers** [reply]
is there any further documentation how to use the ```hero``` block. I assume the intro on your landing page is using the same?
